package.json

```diff
@@ -85,34 +84,33 @@
     "leaflet": "^1.9.4",
     "leaflet.markercluster": "^1.5.3",
     "lodash": "^4.17.21",
-    "monaco-editor": "^0.37.1",
-    "next": "^13.5.4",
-    "next-plausible": "^3.12.0",
+    "monaco-editor": "^0.52.0",
```

Comment:
This could technically be breaking

Reply:
good point — we just use this to render the nice json preview here: https://toiletmap-ey2ebn145-public-convenience-ltd.vercel.app/explorer/loos/fe9b7bb8acad4acee7eafcd8 and all seems well